andela/haven-ah-backend
# #161877976 Add update an article feature

#### What does this PR do?
- Add Update an article feature

#### Description of Task to be completed?
- Create the route for updating an article
- Write an integration test for route
- Write documentation

#### How should this be manually tested?
- git clone `https://github.com/andela/haven-ah-backend.git`.
- Install `node` if not installed.
- ensure that PostgreSQL is running on your local machine.
- run `npm install` to install the npm packages.
- run `npm test` to run the unit and integration tests.

To test on Postman
- run `npm run start-dev`
- Add the title, body and description as `x-www-form-urlencoded`
- On postman run PATCH `localhost:5000/api/articles/:slug/` (set JWT token as`x-access-token` in header).
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161877976

#### Screenshots (if appropriate)

Good Response:
<img width="1063" alt="screen shot 2018-11-18 at 6 21 49 am" src="https://user-images.githubusercontent.com/31534129/48669004-5c88e880-eafa-11e8-88ef-359d7eda25d7.png">
